.github/workflows/test-k8s-lib-injection.yaml

```diff
@@ -8,69 +8,67 @@ on:
     - cron: '0 4 * * *'
 
 jobs:
-  lib-injection-test:
+  injection-tests-nodejs:
```

Comment:
I decided not to remove this file...but another option it is moving these jobs to system-tests.yml
what do you think?